raven_aiohttp.py

```diff
@@ -19,30 +19,38 @@ class AioHttpTransport(AsyncTransport, HTTPTransport):
     def __init__(self, parsed_url, *, verify_ssl=True, resolve=True,
                  timeout=defaults.TIMEOUT,
                  keepalive=True, family=socket.AF_INET, loop=None):
+        self.verify_ssl = verify_ssl
+        self.resolve = resolve
```

Comment:
@asvetlov 
I have added properties. 
Probably it's as best compromise, how do you think?